### PR TITLE
Use Pulumi to set up an EC2 instance

### PR DESCRIPTION
**Pull request by guy9050**

### CODE REVIEW

File: pulumi/Pulumi.dev.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+config:
+  aws:region: us-east-1
```
Comment (lmilbaum) [inline pulumi/Pulumi.dev.yaml]: Can you please switch to `eu-west-2`. This is our default region.

File: pulumi/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+pulumi>=3.0.0,<4.0.0
```
Comment (lmilbaum) [inline pulumi/requirements.txt]: Please pin specific package versions. It is required for a reproducible build. Same goes to the next line.

**Review by lmilbaum — CHANGES_REQUESTED** [review]
Please add a make target and add some notes in the README

File: pulumi/Pulumi.dev.yaml (hunk 1 above)
Comment (lmilbaum) [reply pulumi/Pulumi.dev.yaml]: Was this changed?

**Comment by lmilbaum** [comment]
> Please add a make target and add some notes in the README

I can't find the make target.

File: pulumi/Pulumi.dev.yaml (hunk 1 above)
Comment (guy9050) [reply pulumi/Pulumi.dev.yaml]: Now it's changed :)

**Comment by guy9050** [comment]
> > Please add a make target and add some notes in the README
> 
> I can't find the make target.

Please see now

File: pulumi/Pulumi.dev.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+config:
+  aws:region: us-east-2
```
Comment (lmilbaum) [inline pulumi/Pulumi.dev.yaml]: I still see `us-east-2` instead of `eu-west-2`

File: pulumi/Pulumi.dev.yaml [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+  crc-pulumi:instance_ami: ami-0521a1ab6cb98215d
+  crc-pulumi:instance_type: t2.micro
+  crc-pulumi:key_name: pulumi-crc-key-pair
+  crc-pulumi:region: us-east-2
```
Comment (lmilbaum) [inline pulumi/Pulumi.dev.yaml]: Here too. Is there a need for duplicate region keys?

**Review by lmilbaum — CHANGES_REQUESTED** [review]

File: pulumi/Pulumi.dev.yaml (hunk 4 above)
Comment (guy9050) [reply pulumi/Pulumi.dev.yaml]: Indeed not needed as of now

**Review by lmilbaum — CHANGES_REQUESTED** [review]
```
[lmilbaum@lmilbaum poc]$ make pulumi-up
cd pulumi && pulumi up -s dev -y && cd -
The stack 'dev' does not exist.

If you would like to create this stack now, please press <ENTER>, otherwise press ^C: 
Created stack 'dev'
Previewing update (dev)

View Live: https://app.pulumi.com/lmilbaum/crc-pulumi/dev/previews/4526a7aa-ab84-4f9d-9b49-44527b17f98f

     Type                 Name            Plan       Info
 +   pulumi:pulumi:Stack  crc-pulumi-dev  create     1 error


Diagnostics:
  pulumi:pulumi:Stack (crc-pulumi-dev):
    error: Program failed with an unhandled exception:
    Traceback (most recent call last):
      File "/home/lmilbaum/repos/github.com/platform-engineering-org/poc/pulumi/./__main__.py", line 4, in <module>
        import pulumi_tls as tls
    ModuleNotFoundError: No module named 'pulumi_tls'


warning: A new version of Pulumi is available. To upgrade from version '3.55.0' to '3.58.0', run 
   $ curl -sSL https://get.pulumi.com | sh
or visit https://pulumi.com/docs/reference/install/ for manual instructions and release notes.
make: *** [Makefile:108: pulumi-up] Error 255
```

**Comment by guy9050** [comment]
Interesting,  for me it's working fine. Can you please update your Pulumi version by running `curl -sSL https://get.pulumi.com | sh`?

Also I have just added `pulumi_tls` into the requirements, so make sure to update your local branch.

**Review by lmilbaum — APPROVED** [review]
LGTM

**Pull request merged**